tests/drivers/can/api_additional/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(can_add)
+
+target_include_directories(app PRIVATE ../../../../../zephyr/tests/drivers/can/api/src/)
```

Comment:
Please use $ENV{ZEPHYR_BASE} to create path, like here: https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/current_consumption/system_off/CMakeLists.txt#L11C28-L11C45

Reply:
just use `${ZEPHYR_BASE}`, no env, that is wrong